schema/src/lib.rs

```diff
@@ -502,10 +502,12 @@ pub enum StringEncoding {
     Utf16,
 }
 
-/// The stable identity of a node, unique within the node's tree.
 // This is NonZeroU64 because we regularly store Option<NodeId>.
+pub type NodeIdContent = std::num::NonZeroU64;
+
+/// The stable identity of a node, unique within the node's tree.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, Serialize, Deserialize, JsonSchema)]
-pub struct NodeId(pub std::num::NonZeroU64);
```

Comment:
Is this change necessary? `NonZeroU64` and `NodeId` should take the same amount of memory and a tuple struct does not modify the memory layout of the underlying type, right?